File: arrays_and_hashing/problems/1295_find_numbers_with_even_number_of_digits.py
'''
Given an array nums of integers, return how many of them contain an even number of digits.
'''

class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        ans = 0

        for n in nums:
            if len(str(n)) % 2:
                continue
            ans += 1
        return ans

File: typescript/problems/1295_find_numbers_with_even_number_of_digits.ts
/**
  Given an array nums of integers, return how many of them contain an even number of digits.
 */

function findNumbers(nums: number[]): number {
    let ans: number = 0;
    for (const n of nums) {
        if (String(n).length % 2 === 0) {
            ans++
        }
    }
    return ans
};